_studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer.cpp

```diff
@@ -81,7 +81,7 @@ Packer * Packer::CreatePacker(UMC::VideoAccelerator * va)
 {
     Packer * packer = 0;
 
-        packer = new PackerVA(va);
+    packer = new PackerVA(va);
```

Comment:
looks like you don't need tmp variable at all

Reply:
Fixed